### PR TITLE
chore: use escape double quotation marks for windows

### PR DESCRIPTION
**Pull request by Mini256**

### CODE REVIEW

**Comment by Mini256** [comment]
PTAL~ @hi-rustin

**Comment by hi-rustin** [comment]
Could you please add the windows ci?

**Comment by Mini256** [comment]
> Could you please add the windows ci?

ok, I will try to add it tomorrow.

**Comment by hi-rustin** [comment]
The CI broken.

**Comment by hi-rustin** [comment]
@Mini256  Please revert your last commit. I will try to fix the CI issue.

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -19,7 +19,7 @@ jobs:
     strategy:
       matrix:
         node-version: [ 10.x, 12.x ]
-        os: [ ubuntu-latest, macos-latest ] # FIXME: prettier can not working on windows.
```
Comment (hi-rustin) [inline .github/workflows/test.yml:22]: Please revert the FIXME.

**Review by hi-rustin — CHANGES_REQUESTED** [review]

File: .github/workflows/test.yml (hunk 1 above)
Comment (Mini256) [reply .github/workflows/test.yml:22]: ok

**Review by hi-rustin — APPROVED** [review]
Good Job!

**Comment by hi-rustin** [comment]
/hold cancel

Windows CI still have issue, maybe fix it in the future.

**Pull request merged**